setup.py

```diff
@@ -71,14 +70,14 @@
     'invenio-mail>=1.0.2,<1.1.0',
     'invenio-rest>=1.1.0,<1.2.0',
     # From auth bundle
-    'invenio-accounts==1.1.1',  # FIXME
     'invenio-oauth2server>=1.0.3,<1.1.0',
     'invenio-oauthclient>=1.1.2,<1.2.0',
     'invenio-userprofiles>=1.0.1,<1.1.0',
     # Not included in previous dependencies
     'invenio-db[postgresql]>=1.0.3,<1.1.0',
     # Invenio dependencies missing from Invenio packages setup.py
     'jsonpatch>=1.23,<2',
+    'six>=1.12.0',
```

Comment:
Why do we need this now?

Reply:
https://travis-ci.org/github/reanahub/reana-server/builds/664785079?utm_medium=notification&utm_source=github_status